pkg/localvolume/localvolume.go

```diff
@@ -0,0 +1,128 @@
+package localvolume
+
+import (
+	"context"
+	"errors"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	core "k8s.io/api/core/v1"
+	//storagev1 "k8s.io/api/storage/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type LocalVolumes interface {
```

Comment:
since package is localvolume I would use names like Volumes, Volume (local is redundant) and I would change the name of the file accordingly

Reply:
removed `local` prefix